src/utilities/validateNextState.js

```diff
@@ -1,5 +1,5 @@
 export default (nextState, reducerName: string, action: Object): null => {
-    if (nextState === undefined) {
```

Comment:
Keep `nextState === undefined`. There is no need to use `typeof` here since we know that the variable does exist. We are only checking whether its been passed a value other than undefined.